senders/slack/slack.go

```diff
@@ -64,15 +63,15 @@ func (sender *Sender) Init(senderSettings interface{}, logger moira.Logger, loca
 	if err != nil {
 		return fmt.Errorf("failed to decode senderSettings to slack config: %w", err)
 	}
-	apiToken := s.APIToken
-	if apiToken == "" {
+
+	if s.APIToken == "" {
```

Comment:
Maybe we should use longer variable names in each sender

Reply:
ok, changed it